wpilibNewCommands/src/test/native/cpp/frc2/command/ProxyScheduleCommandTest.cpp

```diff
@@ -17,9 +17,7 @@ TEST_F(ProxyScheduleCommandTest, ProxyScheduleCommandSchedule) {
 
   bool scheduled = false;
 
-  InstantCommand toSchedule([&scheduled] { scheduled = true; }, {});
-
-  ProxyScheduleCommand command(&toSchedule);
+  ProxyScheduleCommand command(std::make_unique<InstantCommand>([&scheduled] { scheduled = true; }, {}));
```

Comment:
Can we add another test case to this file instead of removing the existing one? I think both ways of using the command are appropriate (having the user own the command and having the ProxyCommand own the command) so we should test both.